tools/deploy

```diff
@@ -41,7 +41,7 @@ upload_chrome_prod_ext() {
 sign_firefox_ext() {
   FIREFOX_STAGE_EXT_ID="{b441de5f-18e6-40ad-a8c2-f1bd2d42cb01}"
   rm -rf dist/firefox-stage
-  unzip dist/*-firefox-stage.xpi -d dist/firefox-stage
+  unzip -q dist/*-firefox-stage.xpi -d dist/firefox-stage
```

Comment:
`unzip` prints the filename of every file it unpacks by default. This is just annoying.